content/docs-beta/SUSHI/running/_index.md

```diff
@@ -0,0 +1,149 @@
+
+While SUSHI is running, it will print status messages as it processes your project files. When SUSHI has completed, you should receive a summary like the following:
+
+```text
+╔════════════════════════ SUSHI RESULTS ══════════════════════════╗
+║ ╭──────────┬────────────┬───────────┬─────────────┬───────────╮ ║
+║ │ Profiles │ Extensions │ ValueSets │ CodeSystems │ Instances │ ║
+║ ├──────────┼────────────┼───────────┼─────────────┼───────────┤ ║
+║ │    1     │     1      │     1     │      1      │     1     │ ║
+║ ╰──────────┴────────────┴───────────┴─────────────┴───────────╯ ║
+║                                                                 ║
+╠═════════════════════════════════════════════════════════════════╣
+║ O-fish-ally error free!                0 Errors      0 Warnings ║
+╚═════════════════════════════════════════════════════════════════╝
+```
+
+### Error Messages
+
+In the process of developing your IG using FSH, you may encounter SUSHI error messages (written to the command console). Most error messages point to a specific line or lines in a `.fsh` file. If possible, SUSHI will continue, despite errors, to produce FHIR artifacts, but those artifacts may omit problematic rules. SUSHI should always exit gracefully. If SUSHI crashes, please report the issue using the [SUSHI issue tracker](https://github.com/FHIR/sushi/issues).
+
+Here are some general tips for debugging:
+
+* **Parsing (syntax) errors should be fixed first.** A single syntax error can ballooon into many other errors, so you should always eliminate syntax errors first. Syntax error messages may include `extraneous input {x} expecting {y}`, `mismatched input {x} expecting {y}` and `no viable alternative at {x}`. These messages indicate that the line in question is not a valid FSH statement.
+* **The order of keywords matters.** The declarations must start with the type of item you are creating (e.g., Profile, Instance, ValueSet).
+* **The order of rules usually doesn't matter, but there are exceptions.** Slices and extensions must be created before they are constrained.
+* **Rules must contain valid paths.** The `No element found at path` error means that although the overall grammar of the rule may be correct, SUSHI could not find the FHIR element you are referring to in the rule. Make sure there are no spelling errors, the element names in the path are correct, and you are using the [path grammar](https://build.fhir.org/ig/HL7/fhir-shorthand/reference.html#fsh-paths) correctly.
+* **The community can help.** If you are getting an error you can't resolve, you can ask for help on the [#shorthand chat channel](https://chat.fhir.org/#narrow/stream/215610-shorthand).
+
+
+## SUSHI Outputs
+
+Based on the inputs in FSH files, **sushi-config.yaml**, and the IG project directory, SUSHI populates the output directory. For example, the customized-ig project from the [Project Structure](/docs-beta/sushi/project/) section would result in output like the following:
+
```

Comment:
Can we update the wording a bit to be more clear that the SUSHI output is _only_ in `fsh-generated`?  All other files are part of the original project source.